queue_job/jobrunner/channels.py

```diff
@@ -193,18 +192,36 @@ def __eq__(self, other):
     def __hash__(self):
         return id(self)
 
+    def set_no_eta(self):
+        self._sorting_key = (None,) + self._sorting_key[1:]
+
+    @property
+    def seq(self):
+        return self._sorting_key[3]
+
+    @property
+    def date_created(self):
+        return self._sorting_key[2]
+
+    @property
+    def priority(self):
+        return self._sorting_key[1]
+
+    @property
+    def eta(self):
+        return self._sorting_key[0]
```

Comment:
```suggestion
        return self._sorting_key.eta
```

Would a namedtuple work for what you want to achieve? It would help readability.